src/main/fc/core.c

```diff
@@ -1270,8 +1270,9 @@ FAST_CODE bool pidLoopReady(void)
 
 FAST_CODE void taskFiltering(timeUs_t currentTimeUs)
 {
-    gyroFiltering(currentTimeUs);
+    updateDshotTelemetry();  // decode and update Dshot telemetry
```

Comment:
Not sure about this. Does this assume DShot being used? What if another ESC protocol is used?

Reply:
Maybe this should be inside `#ifdef USE_DSHOT_TELEMETRY`?

Reply:
Good catch